Description: Update ICU test as Debian/Ubuntu ship ICU
 4.4 not 4.6/4.8 as used by upstream - this alters the
 collation key order slightly
Origin: https://github.com/apache/lucene-solr/commit/694ec2f6589ddefa1828437621f4dda7e827ae47#diff-18
Author: James Page <james.page@ubuntu.com>
Forwarded: not-needed

Index: lucene-solr/lucene/contrib/icu/src/test/org/apache/lucene/collation/TestICUCollationKeyFilter.java
===================================================================
--- lucene-solr.orig/lucene/contrib/icu/src/test/org/apache/lucene/collation/TestICUCollationKeyFilter.java	2012-05-28 10:05:16.627582079 +0100
+++ lucene-solr/lucene/contrib/icu/src/test/org/apache/lucene/collation/TestICUCollationKeyFilter.java	2012-05-28 10:49:56.000000000 +0100
@@ -94,6 +94,6 @@
     // orderings - "BFJHD" is the ordering for the ICU Collator for Locale.US.
     testCollationKeySort
     (usAnalyzer, franceAnalyzer, swedenAnalyzer, denmarkAnalyzer, 
-     "BFJHD", "ECAGI", "BJDFH", "BJDHF");
+     "BFJHD", "EACGI", "BJDFH", "BJDHF");
   }
 }
Index: lucene-solr/lucene/contrib/icu/src/test/org/apache/lucene/collation/TestICUCollationKeyAnalyzer.java
===================================================================
--- lucene-solr.orig/lucene/contrib/icu/src/test/org/apache/lucene/collation/TestICUCollationKeyAnalyzer.java	2012-05-23 09:39:25.597827260 +0100
+++ lucene-solr/lucene/contrib/icu/src/test/org/apache/lucene/collation/TestICUCollationKeyAnalyzer.java	2012-05-28 11:00:20.028209522 +0100
@@ -76,7 +76,7 @@
     // orderings - "BFJHD" is the ordering for the ICU Collator for Locale.US.
     testCollationKeySort
     (usAnalyzer, franceAnalyzer, swedenAnalyzer, denmarkAnalyzer, 
-     "BFJHD", "ECAGI", "BJDFH", "BJDHF");
+     "BFJHD", "EACGI", "BJDFH", "BJDHF");
   }
   
   public void testThreadSafe() throws Exception {
